pylint/checkers/utils.py

```diff
@@ -2190,3 +2193,19 @@ def is_terminating_func(node: nodes.Call) -> bool:
         pass
 
     return False
+
+
+def get_properties(config: argparse.Namespace) -> tuple[set[str], set[str]]:
+    """Returns a tuple of property classes and names.
+
+    Property classes are fully qualified, such as 'abc.abstractproperty' and
+    property names are the actual names, such as 'abstract_property'.
+    """
+    property_classes = {BUILTIN_PROPERTY}
```

Comment:
```suggestion
    property_classes = {"builtins.property"}
```
This is never going to change, we might as well make the code easier to read.